repos/system_upgrade/el7toel8/models/grubdevice.py

```diff
@@ -1,13 +1,22 @@
 from leapp.models import Model, fields
 from leapp.topics import SystemFactsTopic
+from leapp.utils.deprecation import deprecated
 
 
+@deprecated(
+    since="2020-09-01",
+    message=("The model is deprecated as we moved the GRUB device detection into library")
```

Comment:
Maybe extend the message with the advice what the developer should do

Reply:
to use the library :) But I will expand the message for the completeness sake...